### PR TITLE
Hide units on machines for hidden services

### PR DESCRIPTION
**Pull request by huwshimi**
Hide units in the machine view if they've been hidden in the added services list. Fixes https://github.com/juju/juju-gui/issues/1118

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4018/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4029/
Test PASSed.

File: jujugui/static/gui/src/app/components/machine-view/machine/test-machine.js [hunk 1]
```diff
@@ -21,12 +21,33 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
...
+  var services;
 
   beforeAll(function(done) {
     // By loading this file it adds the component to the juju components.
     YUI().use('machine-view-machine', function() { done(); });
   });
 
+  beforeEach(function () {
+    services = {
+      getById: sinon.stub().returns({
```
Comment (kadams54) [inline jujugui/static/gui/src/app/components/machine-view/machine/test-machine.js]: Thanks for pulling this setup out of all the various test methods.

**Comment by kadams54** [comment]
QA is :ok:, code is :+1: - thanks!

**Comment by kadams54** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**